run_unittests.py

```diff
@@ -484,7 +484,7 @@ def setUp(self):
         src_root = os.path.join(os.getcwd(), src_root)
         self.src_root = src_root
         self.prefix = '/usr'
-        self.libdir = os.path.join(self.prefix, 'lib')
+        self.libdir = 'lib'
```

Comment:
Is this save? Does any other test use self.libdir?

Reply:
It is safe w.r.t. passing to meson. Other tests do use `self.libdir` and those have been changed in the same commit.